stable/ignite/Chart.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+appVersion: "1.0"
```

Comment:
From the image tag it looks like this should be 2.6.0... Is that the case or is the image tag version always going to be different from the app version?

Reply:
I think it's better to delete it, cause we want ignite version to be configurable through `values.yaml` so `appVersion` is unnecessary.

Reply:
appVersion is actually used as metadata for things like hub.kubeapps.com. Its useful to have it be meaningful.

Reply:
Got it, thanks! I've changed appVersion to 2.6.0.